### PR TITLE
Fix namespace

### PR DESCRIPTION
**Pull request by kbaba1001**
Because I couldn't deploy to clojars, I fixed group name and gpg-key setting.

### CODE REVIEW

File: lein-metabill/project.clj [hunk 1]
```diff
@@ -1,8 +1,7 @@
-(defproject xcoo/lein-metabill "0.1.0"
+(defproject jp.xcoo/lein-metabill "0.1.1-SNAPSHOT"
   :description "Read build meta info"
   :url "https://github.com/xcoo/metabill"
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0.html"}
-  :dependencies [[xcoo/metabill "0.1.0"]]
```
Comment (kbaba1001) [inline lein-metabill/project.clj:6]: I had forgot to fix this line, so I fixed it and updated lein-metabill version.

File: lein-metabill/project.clj [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
-  :eval-in-leiningen true
-  :signing {:gpg-key "developer@xcoo.jp"})
```
Comment (federkasten) [inline lein-metabill/project.clj:8]: Please revert gpg signing :thinking: This is a very important setting.

**Review by federkasten — CHANGES_REQUESTED** [review]
Please fix the following two points in the README;

1. Wrong namespace `xcoo/` is remaining.
2. `with-commit-hash` usage description (with screenshot) is missing.

In addition, as a comment, I think it would be more helpful to output `metabill.edn` in `target/` or `resources/` where the classpaths are active, rather than in the top directory. Is there any reason to put it in the top directory? In this case, users need to manually add `metabill.edn` to classpaths for production builds.

**Review by federkasten — APPROVED** [review]
Merge once. I'll make some additional changes on my end :ok_man:

**Pull request merged**